nb/api/noOp.md

```diff
@@ -0,0 +1,13 @@
+### noOp()
```

Comment:
No complaining, just wondering. Why do we need this?

Reply:
Oh, I think I get it. It can be passed in place of a function which does something when passing an operation is required?

Reply:
Yeah.